[Outlook] (spam reporting) Document ribbon behavior (#4711)
OfficeDev/office-js-docs-pr: docs/outlook/spam-reporting.md

As you develop and test the integrated spam-reporting feature in your add-in, be mindful of its characteristics and limitations.

> - In Outlook on the web and on Windows (new and classic), an integrated spam-reporting add-in replaces the native **Report** button in the Outlook ribbon. If multiple spam-reporting add-ins are installed, they will all appear in this location on the ribbon.

Reviewer: ```suggestion
- In Outlook on the web and on Windows (new and classic), an integrated spam-reporting add-in replaces the native **Report** button in the Outlook ribbon. If multiple spam-reporting add-ins are installed, they will all appear in the **Report** section of the ribbon.
```
